tests/test_cases.cpp

```diff
@@ -420,6 +467,16 @@ TEST(TextManip, to_upper)
     EXPECT_EQ("HELLO STRUTIL", strutil::to_upper("HeLlo StRUTIL"));
 }
 
+TEST(TextManip, to_capitalize)
+{
+    EXPECT_EQ("HeLlo StRUTIL", strutil::to_capitalize("heLlo StRUTIL"));

```

Comment:
Add another test with the input string: `std::string("+ is an operator.")` and expected output: `std::string("+ is an operator.")`.